server/src/main/java/com/sauron/model/entities/Account.java

```diff
@@ -0,0 +1,64 @@
+package com.sauron.model.entities;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "ACCOUNTS")
```

Comment:
I would rearrange this entity. Obviously mapping user-bank is required but other attributes can be deleted. They won't live longer then the session so there is no point in keeping them in db.

Reply:
I was thinking about balance as a temporary placeholder, from which we could get the balance for Rest endpoint. But ok, removing it for now.

Reply:
Also, it's up for discution whether or not we want to store information about current balance and last transaction on our side in case of no connectivity or something like this.